azure/config.go

```diff
@@ -120,6 +118,14 @@ func getAccount(cfg stow.Config) (account, key string, baseUrl string, APIVersio
 	return acc, key, baseUrl, APIVersion, useHTTPS, nil
 }
 
+func GetBaseAzureUrlOrDefault(cfg stow.Config) string {
```

Comment:
this function doesn't have to be exported, right? (Since all invocations of it are in the same package)

Reply:
Good point @eapolinario, fixed